ledger/participant-integration-api/src/main/scala/platform/store/dao/LedgerDao.scala

```diff
@@ -102,19 +93,12 @@ private[platform] trait LedgerReadDao extends ReportsHealth {
   /** Looks up the current ledger end */
   def lookupLedgerEnd()(implicit loggingContext: LoggingContext): Future[Offset]
 
+  /** Looks up the current ledger end sequential id */
+  def lookupLedgerEndSequentialId()(implicit loggingContext: LoggingContext): Future[Long]
```

Comment:
I am wondering about granularity needes vs. LedgerReadDao surface size: would it make sense just to add this info the lookupLedgerEnd method?